src/libutils/oglapphelpers/oglapp.h

```diff
@@ -113,6 +118,10 @@ class OglApp
     // Helper to print GL info.
     void virtual printGLInfo() const noexcept;
 
+    // Return a pointer of either ScreenApp or HeadlessApp depending on the
+    // OCIO_HEADLESS_ENABLED preprocessor
+    static OglAppRcPtr createOglAppPtr(const char * winTitle, int winWidth, int winHeight);
```

Comment:
Forgot to mention that any static method starts with an upper case and that the returned type does not add anything to understand the method purpose. So, the method name should be `CreateOglApp()`